### DISCUSSION TITLE
IPA source recommend

### DISCUSSION THREAD

**Discussion by ResistanceTo** [PlayCover/PlayCover#617] [Q&A] [answered]
Expect to get the original ipa file download source

**Comment by amirsaam** [comment, marked as answer]
~You can either use my source
`https://amrsm.ir/decrypted.json`
or~ (currently out of service)
use https://decrypt.day

**Reply by ResistanceTo** [reply]
Thanks, this is useful!

**Comment by TylerAustin7** [comment]
https://amrsm.ir/decrypted.json is not working
